src/style.css

```diff
@@ -0,0 +1,73 @@
+@font-face {
+  font-family: "Avenir";
+  src: url("fonts/Avenir.ttc");
+  font-style: normal;
+  font-weight: normal;
+}
+
+@font-face {
+  font-family: "Avenir-Book";
+  src: url("fonts/Avenir-Book.ttf");
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: "Avenir-Heavy";
+  src: url("fonts/Avenir-Heavy.ttf");
+  font-weight: 900;
+}
+
+* {
+  box-sizing: border-box;
+}
+
+.bigSearchBar {
+  margin-top: 20px;
+}
+
+.smallSearchBar {
+  margin-top: 20px;
+}
+
+.search__input {
+  font-family: "Avenir-Heavy", Helvetica, sans-serif;
+  font-size: 16px;
+  width: 100%;
+  height: 70px;
+  background: url("images/Search.svg");
+  background-repeat: no-repeat;
+  background-position: 26px 25px;
+  text-indent: 62px;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+}
+
+.smallSearch__input {
+  font-size: 14px;
+  height: 42px;
+  background-size: 11px 11px;
+  background-position: 12px 14px;
+  text-indent: 33px;
+
+}
+
+.search__input:hover {
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+}
+
+.search__input:focus {
+  font-family: "Avenir-Heavy", Helvetica, sans-serif;
+  border-radius: 3px;
+  background-color: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+}
```

Comment:
Right now you have blue outline on focus, you should remove it
![image](https://user-images.githubusercontent.com/37161315/136983113-3dad4730-3f0f-4694-af7c-25ecb13160e6.png)